cmd/nerdctl/pull.go

```diff
@@ -96,7 +107,16 @@ func pullAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	verifier, err := cmd.Flags().GetString("verify")
+	if err != nil {
+		return err
+	}
+
 	if scheme, ref, err := referenceutil.ParseIPFSRefWithScheme(args[0]); err == nil {
+		if verifier != "none" {
+			return err
```

Comment:
`err` seems nil here. we should return an error something like `--verify flag is not supported on IPFS as of now`

Reply:
done